var/spack/repos/builtin/packages/tut/package.py

```diff
@@ -25,17 +25,19 @@
 from spack import *
 
 
-class Tut(Package):
+class Tut(WafPackage):
```

Comment:
And you didn't try how fun it is doing the machinery with metaclasses :laughing: